packages/flutter/lib/src/material/raised_button.dart

```diff
@@ -360,27 +404,42 @@ class RaisedButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final ThemeData theme = Theme.of(context);
-    final ButtonThemeData buttonTheme = ButtonTheme.of(context);
-    final Color fillColor = _getFillColor(theme, buttonTheme);
-    final Color textColor = _getTextColor(theme, buttonTheme, fillColor);
-
-    return new RawMaterialButton(
-      onPressed: onPressed,
-      onHighlightChanged: onHighlightChanged,
-      fillColor: fillColor,
-      textStyle: theme.textTheme.button.copyWith(color: textColor),
-      highlightColor: _getHighlightColor(theme, buttonTheme),
-      splashColor: splashColor ?? theme.splashColor,
-      elevation: elevation,
-      highlightElevation: highlightElevation,
-      disabledElevation: disabledElevation,
-      padding: padding ?? buttonTheme.padding,
-      constraints: buttonTheme.constraints,
-      shape: shape ?? buttonTheme.shape,
-      animationDuration: animationDuration,
-      child: child,
+    return new PlatformBuilder(
+      materialWidgetBuilder: (BuildContext context) {
+        final ThemeData theme = Theme.of(context);
+        final ButtonThemeData buttonTheme = ButtonTheme.of(context);
+        final Color fillColor = _getFillColor(theme, buttonTheme);
+        final Color textColor = _getTextColor(theme, buttonTheme, fillColor);
+
+        return new RawMaterialButton(
```

Comment:
hmm, i thought we had changed to using MaterialButton to share more code. cc @HansMuller

Reply:
We haven't done that yet.